crates/sp-domains-fraud-proof/src/host_functions.rs

```diff
@@ -263,7 +263,7 @@ where
     ) -> Option<DomainInherentExtrinsic> {
         let DomainInherentExtrinsicData {
             // Used by caller
-            block_randomness: _,
```

Comment:
This field was introduced recently and is not exist in the consensus runtime in mainnet, so we should also remove it

Reply:
Specifically in #3312, where it wasn’t caught in review. Is there an automated test we could add to catch accidental changes like this?

Reply:
It is hard to test compatibility with an existing network in unit/integration test, usually we will test runtime upgrade before release (by simulating the existing network in a local network).